spec/sec-properties-of-the-regexp-prototype-object-patch.html

```diff
@@ -69,29 +71,28 @@ <h1>Runtime Semantics: RegExpBuiltinExec ( _R_, _S_ )</h1>
           1. <del>Else _fullUnicode_ is *false*,</del>
             1. <del>Assert: _captureI_ is a List of code units.</del>
             1. <del>Let _capturedValue_ be the String value consisting of the code units of _captureI_.</del>
-          1. <ins>If the _i_th capture of _R_ was defined with a |GroupName|, then</ins>
-            1. <ins>Let _capturedGroup_ be the StringValue of the corresponding |RegExpIdentifierName|.</ins>
-          1. <ins>Else,</ins>
-            1. <ins>Let _capturedGroup_ be *undefined*.</ins>
           1. <ins>If _captureI_ is *undefined*, then</ins>
             1. <ins>Let _capturedValue_ be *undefined*.</ins>
-            1. <ins>Add *undefined* as the last element of _indices_.[[Indices]].</ins>
+            1. <ins>Add *undefined* as the last element of _indices_.</ins>
           1. <ins>Else,</ins>
             1. <ins>Let _captureStart_ be _captureI_'s _startIndex_.</ins>
             1. <ins>Let _captureEnd_ be _captureI_'s _endIndex_.</ins>
             1. <ins>If _fullUnicode_ is *true*, then</ins>
               1. <ins>Set _captureStart_ to ! GetStringIndex(_S_, _Input_, _captureStart_).</ins>
               1. <ins>Set _captureEnd_ to ! GetStringIndex(_S_, _Input_, _captureEnd_).</ins>
-            1. <ins>Let _capture_ be Match { [[StartIndex]]: _captureStart_, [[EndIndex]:: _captureEnd_, [[GroupName]]: _capturedGroup_ }.</ins>
-            1. <ins>Add _capture_ as the last element of _indices_.[[Indices]].</ins>
+            1. <ins>Let _capture_ be the Match { [[StartIndex]]: _captureStart_, [[EndIndex]:: _captureEnd_ }.</ins>
+            1. <ins>Add _capture_ as the last element of _indices_.</ins>
```

Comment:
```suggestion
            1. <ins>Append _capture_ to _indices_.</ins>
```
?

Reply:
"Add _x_ as the last element of _y_" is used in 10 places in the spec today. "Append _x_ as the last element of _y_" is used in 15 places. It doesn't seem like the spec is consistent with the terminology here.

Reply:
In cases like these, we go with whatever is most frequent - in this case, "Append x to y". The others are historical debt we need to clean up.